homeassistant/components/homekit_controller/connection.py

```diff
@@ -361,7 +362,7 @@ async def async_process_entity_map(self) -> None:
         # is especially important for BLE, as the Pairing instance relies on the entity map
         # to map aid/iid to GATT characteristics. So push it to there as well.
 
-        self.pairing.pairing_data["accessories"] = self.accessories
+        self.pairing.pairing_data["accessories"] = self.accessories  # type: ignore[attr-defined]
```

Comment:
I figured an ignore was best for now since we didn't want to type this to IPParing since thats about to change

Reply:
I'd suggest casting to AbstractPairing but I'm not entirely sure I didn't move it in the dev branch!

Reply:
After aiohomekit dev merges, I'm going to revive that typing PR we deferred to avoid future conflict which should fix most of the type ignores we have now.